docker-compose.yml

```diff
@@ -3,13 +3,13 @@ version: "3"
 services:
   telemetry-frontend:
     build:
-      dockerfile: frontend.Dockerfile
-      context: ./
+      dockerfile: Dockerfile
+      context: ./frontend/
     # Copy in changes to the ui, so no need to rebuild the images.
     volumes:
       - /app/node_modules
       - ./packages:/app/packages
-    command: yarn start:frontend
+    # command: yarn start:frontend
```

Comment:
Is this supposed to be commented out? Noticed no CMD directive in the Dockerfile

Reply:
This is from @maciejhirsz 's docker-enhancing commit, he might want to do more cleanup that I can pull in. ( @maciejhirsz I was able to successfully build/run both images locally).

Reply:
It should be removed. Frontend is served from static files via nginx, `yarn start:frontend` runs dev environment.